workshop/posting-errors-guidelines.md

```diff
@@ -78,7 +78,7 @@ For our example error, here's how each of the "four points" (and the optional on
 - Note that line numbers change as you've added code, so line numbers as a reference aren't as helpful.
 
 *For our example:*
-<img src = "screenshots/where-is-the-error.png" width = "600">
+<img src = "screenshots/where-is-the-error.png" alt = "Where is the error" width = "600">
```

Comment:
Missing break means this would be on the same line as the text
```suggestion

<img src = "screenshots/where-is-the-error.png" alt = "Where is the error" width = "600">
```

Reply:
Can I just say how tempting it was to write "What is your favorite color", etc. here?